controllers/managedcluster/managedcluster-controller_test.go

```diff
@@ -28,12 +28,16 @@ func TestReconcile(t *testing.T) {
 	ctx := context.Background()
 
 	c := &ManagedClusterReconciler{
-		Client: clientfake.NewFakeClientWithScheme(testScheme),
+		Client: clientfake.NewClientBuilder().WithScheme(testScheme).Build(),
 		Log:    ctrl.Log.WithName("controllers").WithName("ManagedClusterReconciler"),
 		Scheme: testScheme,
 	}
 
 	if err := c.Client.Create(ctx, &hivev1.ClusterClaim{
+		// TypeMeta: metav1.TypeMeta{
```

Comment:
@fxiang1 @xiangjingli I left this commented out so the test doesn't pass falsely, but I assume we need to define this elsewhere, maybe in the managedcluster-controller.go or clusterclaims-controller? We know the gvk via the hivev1 scheme anyway https://github.com/openshift/hive/blob/b6cdaa9cb31727f22c9e354abc321b7ee0ca4ee6/contrib/pkg/clusterpool/clusterclaim.go#L77